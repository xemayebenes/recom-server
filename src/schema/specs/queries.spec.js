import Queries from '../queries';

describe('Queries', () => {
    it('should match snapshot', () => {
        expect(Queries).toMatchSnapshot();
    });

});