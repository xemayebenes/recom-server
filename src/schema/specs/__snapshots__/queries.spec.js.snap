// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Queries should match snapshot 1`] = `
"
type Query {
  getFortuneCookie: String @cacheControl(maxAge: 5)
  searchFilms(searchText: String!, language: String): [SearchMovieItem]
  searchSeries(searchText: String!, language: String): [SearchSerieItem]
  getMovie(externalId: Int!, language: String): Movie
  getSerie(externalId: Int!, language: String): Serie @cacheControl(maxAge: 5)
  getUserSeries(userId: String!): [UserSerie]
  getUserMovies(userId: String!): [UserMovie]
  getUserLastItems(userId: String!): [LastItem]
  getUserMovie(userId: String!, id: String!): UserMovie
  getUserSerie(userId: String!, id: String!): UserSerie
}
"
`;
